python/postprocessing/modules/btv/btagSFProducer.py

```diff
@@ -107,7 +119,9 @@ def beginJob(self):
         # (cf. https://twiki.cern.ch/twiki/bin/viewauth/CMS/BTagCalibration )
         self.calibration = ROOT.BTagCalibration(self.algo, os.path.join(self.inputFilePath, self.inputFileName))
         self.readers = {}
-        for wp in [ "L", "M", "T", "shape_corr" ]:
+        #for wp in [ "L", "M", "T", "shape_corr" ]:
+        # CV: shape corrections ('iterativefit' method) not supported for 2017 ReReco data and RunIIFall17 MC yet
+        for wp in [ "L", "M", "T" ]: 
```

Comment:
this is also a problem for 2016 processing, please make it so that it works as it worked before when running on 2016

Reply:
Ok, done.